custodian_templates/config_template.py

```diff
@@ -0,0 +1,20 @@
+from typing import Mapping
+
+
+def custodian_organizations_config_template(config: Mapping) -> Mapping:
+    dict_template = {
+        "accounts": [
+            {
+                "account_id": account["account_id"],
+                "name": account["account_name"],
+                "regions": [region for region in config["aws"]["regions"]],
```

Comment:
Can `[region for region in config["aws"]["regions"]]` just be `config["aws"]["regions"]`?

Reply:
Fixed.